server/src/main/java/com/skillmagnet/Lesson/LessonRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.skillmagnet.Lesson;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface LessonRepository extends JpaRepository<Lesson, Integer> {
```

Comment:
I was going to ask about this, using `<Lesson, Integer>` rather then `<Lesson, Long>`. I like it, avoids having to create `findById(int i)`